linux_pipeline/Jenkinsfile_rhel_kernel_validation

```diff
@@ -172,3 +172,29 @@ stage ("Send Email Report"){
         }
     }
 }
+
+stage ("Trigger Hotfix"){
+    node ("jenkins-meta-slave") {
+        if (!sendMail) {
+            return
+        }
+        def TestDistros = ""
+        dir ("trigger-hotfix-${env.BUILD_NUMBER}-${env.BRANCH_NAME}") {
+            stashList.each() {
+                unstash it
+                VersionLog = readFile("${it}_lis_check.log")
+                if (VersionLog.contains("built-in driver")) {
+                    def Kernelversion = (VersionLog =~ "Running kernel version: (.*).x86_64")[0][1]
+                    TestDistros+="${it}=https://${STORAGE_ACCOUNT}.blob.core.windows.net/kernel/kernel-${Kernelversion}.x86_64.rpm,https://${STORAGE_ACCOUNT}.blob.core.windows.net/kernel/kernel-devel-${Kernelversion}.x86_64.rpm"+"\n"
+                }
+            }
+            if(TestDistros){
+                TestDistros=TestDistros.trim()
+                echo "TRIGGERING the HOT-FIX pipline for ${TestDistros}"
+                build job: 'Azure/Sandbox/pipeline-lis-hotfix', parameters: [ text(name: 'TARGET_LIST', value: "${TestDistros}")],
```

Comment:
Remove sandbox and update it accordingly once the pipeline-lis-hotfix is made available.

Reply:
Updated the build job path to LIS/pipeline-lis-hotfix